src/pvoutput/utils.py

```diff
@@ -54,17 +77,20 @@ def get_rate_limit_header(response_object):
     return retval
 
 
-def validate_data(self, data, apiset):
+def validate_data(data: dict, apiset: dict, donation_made: bool = False) -> bool:
```

Comment:
I understand but since this function is not in the class itself, linting complains about the self argument.
Since only the donation_made variable is read from self, I am in the opinion that it can be moved out of the class.
It does not interact with the class (sync or async) in any other way.
This also makes it more testable, since it is no longer in the class.